### ISSUE TITLE
Capistrano3  deployments should not be blocked by intermittent SSL failures to api.rollbar.com

### ISSUE THREAD

**Issue by jmgarnier** [rollbar/rollbar-gem#982]
Yesterday https://status.rollbar.com/incidents/39rm7hwmplpm  (Intermittent SSL failures when connecting to https://api.rollbar.com ) blocked the deployment of all Rails apps.

I was able to deploy by commenting the following hooks:
https://github.com/rollbar/rollbar-gem/blob/54b32011b4f139db198ae3c32e166580d487492e/lib/rollbar/capistrano3.rb#L39-L46

Stacktrace:
```
SSL_connect returned=1 errno=0 state=error: certificate verify failed (unable to get local issuer certificate) (SSHKit::Runner::ExecuteError)
/var/jenkins_home/.rvm/rubies/ruby-2.6.6/lib/ruby/2.6.0/net/protocol.rb:44:in `connect_nonblock': SSL_connect returned=1 errno=0 state=error: certificate verify failed (unable to get local issuer certificate) (OpenSSL::SSL::SSLError)
	from /var/jenkins_home/.rvm/rubies/ruby-2.6.6/lib/ruby/2.6.0/net/protocol.rb:44:in `ssl_socket_connect'
	from /var/jenkins_home/.rvm/rubies/ruby-2.6.6/lib/ruby/2.6.0/net/http.rb:996:in `connect'
	from /var/jenkins_home/.rvm/rubies/ruby-2.6.6/lib/ruby/2.6.0/net/http.rb:930:in `do_start'
	from /var/jenkins_home/.rvm/rubies/ruby-2.6.6/lib/ruby/2.6.0/net/http.rb:919:in `start'
	from /var/jenkins_home/.rvm/rubies/ruby-2.6.6/lib/ruby/2.6.0/net/http.rb:605:in `start'
	from /var/jenkins_home/.rvm/gems/ruby-2.6.6/gems/rollbar-2.26.0/lib/rollbar/deploy.rb:48:in `send_request'
```

Can you rescue `OpenSSL::SSL::SSLError` / retry when it happens and add a warning maybe?

**Comment by truongduyng** [comment]
Failsafe from rollbar-gem. OpenSSL::SSL::SSLError: "SSL_connect returned=1 errno=0 state=error: certificate verify failed (unable to get local issuer certificate)" in /home/deploy/.rbenv/versions/2.5.5/lib/ruby/2.5.0/net/protocol.rb:44:in `connect_nonblock': error in process_item

me too

**Comment by waltjones** [comment]
@truongduyng Contact support to request a credit for your quota.

**Comment by waltjones** [comment]
Fix released in v2.26.1 https://github.com/rollbar/rollbar-gem/releases/tag/v2.26.1
